examples/xgboost_simple.py

```diff
@@ -19,6 +19,8 @@
 
 # FYI: Objective functions can take additional arguments
 # (https://optuna.readthedocs.io/en/stable/faq.html#objective-func-additional-args).
+
+
```

Comment:
Could you remove these blank lines because the comments above are related to the function?

Reply:
Okay, I'll remove that!! Thanks for your review!!